### PR TITLE
Fix datepicker error

### LINKED ISSUE #218: Datepicker not working on Compare Tab
Datepicker errors out on Comparison tab.

## Expected Behavior
Choosing dates should return filtered races

## Actual Behavior
Datepicker returns error

## Steps to Reproduce

1. Develop branch
2. Compare tab
3. Choose dates
4.

## Your Environment

* Local Mac

### PR DESCRIPTION
**Pull request by tingaloo**
## Description
Fixes date picker bug.
Fixes #218 

## How Has This Been Tested?

Local
Mac OS Sierra

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I've talked through the changes with a team lead
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by mkalish — APPROVED** [review]

File: src/components/dateRange.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import moment from 'moment';
 import React from 'react';
 import DayPicker, { DateUtils } from 'react-day-picker';
 import { Button, Col, ButtonGroup, Row } from 'react-bootstrap';
+import 'react-day-picker/lib/style.css';
```
Comment (romoy) [inline src/components/dateRange.jsx]: could the following be update or replaced https://github.com/codefordc/dc-campaign-finance-watch/blob/develop/src/styles/reactdaypicker.css?

could the component style change move to the style dir?

File: src/components/dateRange.jsx [hunk 2]
```diff
@@ -105,4 +102,5 @@ class DateRangeComponent extends React.Component {
   }
 }
 
+// export default DateRangeComponent;
```
Comment (romoy) [inline src/components/dateRange.jsx]: could line 105 be removed?

File: package.json [hunk 3]
```diff
@@ -96,7 +96,7 @@
     "prop-types": "^15.5.8",
     "react": "^15.5.3",
     "react-bootstrap": "^0.30",
-    "react-day-picker": "^1.3.2",
+    "react-day-picker": "^5.4.3",
```
Comment (romoy) [inline package.json:99]: nice

File: src/components/dateRange.jsx [hunk 4]
```diff
@@ -14,20 +15,16 @@ class DateRangeComponent extends React.Component {
     };
   }
 
-  handleStartDateClick(event, day, modifiers) {
+  handleStartDateClick(day, modifiers) {
     this.setState({
-      startDate: modifiers.indexOf('selected') > -1
-        ? null
-        : moment(day.toISOString())
+      startDate: moment(day.toISOString())
```
Comment (romoy) [inline src/components/dateRange.jsx:19]: nice

File: src/components/dateRange.jsx (hunk 1 above)
Comment (tingaloo) [reply src/components/dateRange.jsx]: Removed, it wasn't changing anything.

**Pull request merged**